### PR TITLE
API Updates

### PR DESCRIPTION
**Pull request by richardm-stripe**
Codegen for openapi c671651.
r? @yejia-stripe
cc @stripe/api-libraries

## Changelog
* Add support for `mandate` on `Charge.payment_method_details.card`
* Add support for `mandate_options` on `PaymentIntentCreateParams.payment_method_options.card`, `PaymentIntentUpdateParams.payment_method_options.card`, `PaymentIntentConfirmParams.payment_method_options.card`, `PaymentIntent.payment_method_options.card`, `SetupIntentCreateParams.payment_method_options.card`, `SetupIntentUpdateParams.payment_method_options.card`, `SetupIntentConfirmParams.payment_method_options.card`, and `SetupIntent.payment_method_options.card`
* Add support for `card_await_notification` on `PaymentIntent.next_action`
* Add support for `customer_notification` on `PaymentIntent.processing.card`
* Change `PaymentLinkCreateParams.line_items` to be required

### CODE REVIEW

File: src/main/java/com/stripe/model/PaymentIntent.java [hunk 1]
```diff
@@ -2248,7 +2336,27 @@ public void setPackage(String package_) {
     @Getter
     @Setter
     @EqualsAndHashCode(callSuper = false)
-    public static class Card extends StripeObject {}
+    public static class Card extends StripeObject {
+      @SerializedName("customer_notification")
+      ProcessingCustomerNotification customerNotification;
```
Comment (richardm-stripe) [inline src/main/java/com/stripe/model/PaymentIntent.java:2341]: :( I think this name needs fixing

File: src/main/java/com/stripe/model/PaymentIntent.java [hunk 2]
```diff
@@ -2248,7 +2336,27 @@ public void setPackage(String package_) {
...
+    }
+  }
+
+  @Getter
+  @Setter
+  @EqualsAndHashCode(callSuper = false)
+  public static class ProcessingCustomerNotification extends StripeObject {
```
Comment (richardm-stripe) [inline src/main/java/com/stripe/model/PaymentIntent.java:2348]: (And this should be nested)

File: src/main/java/com/stripe/model/SetupIntentTypeSpecificPaymentMethodOptionsClient.java [hunk 3]
```diff
@@ -0,0 +1,11 @@
+// File generated from our OpenAPI spec
```
Comment (richardm-stripe) [inline src/main/java/com/stripe/model/SetupIntentTypeSpecificPaymentMethodOptionsClient.java:1]: Gotta delete this.

**Comment by kamil-stripe** [comment]
re-r? @yejia-stripe

**Review by kamil-stripe — APPROVED** [review]

**Pull request merged**